Extension/ui/settings.ts

```diff
@@ -270,6 +271,7 @@ class SettingsApp {
             (<HTMLInputElement>document.getElementById(elementId.compileCommands)).value = config.compileCommands ? config.compileCommands : "";
             (<HTMLInputElement>document.getElementById(elementId.configurationProvider)).value = config.configurationProvider ? config.configurationProvider : "";
             (<HTMLInputElement>document.getElementById(elementId.forcedInclude)).value = joinEntries(config.forcedInclude);
+            (<HTMLInputElement>document.getElementById(elementId.dotConfig)).value = joinEntries(config.dotConfig);

```

Comment:
Since `config.dotConfig` will only have one file path, this does not need `joinEntries`.

For setting elements with one value:
```
(<HTMLInputElement>document.getElementById(elementId.dotConfig)).value = config.dotConfig ? config.dotConfig : "";
```